deploy/olm-catalog/akka-cluster-operator/0.0.1/akka-cluster-operator-certified.v0.0.1.clusterserviceversion.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
     capabilities: Seamless Upgrades
     categories: Application Runtime
     certified: 'false'
-    containerImage: 'lightbend-docker-testing-registry.bintray.io/akka-cluster-operator-certified:0.0.4'
+    containerImage: 'registry.connect.redhat.com/lightbend/akka-cluster-operator:latest'
```

Comment:
Do you want that to be 
registry.connect.redhat.com/lightbend/akka-cluster-operator **-certified** :latest

Reply:
Good question. 
No. Here's why:
![image](https://user-images.githubusercontent.com/787107/66968860-de033680-f03b-11e9-96ed-4228d46ecb60.png)

Reply:
I see.  So the
> `docker push lightbend-docker-registry.bintray.io/lightbend/akkacluster-operator-certified:latest`

in `push.yml` is just so we have our own copy?

Reply:
Good question. I've setup the build config in the certified project setup. Maybe better to just remove this publish target.

Reply:
Removed push.yml.